receiver/gitlabreceiver/config.go

```diff
@@ -30,6 +30,8 @@ const (
 	defaultGitlabEventHeader       = "X-Gitlab-Event"
 	defaultGitlabEventUUIDHeader   = "X-Gitlab-Event-UUID"
 	defaultIdempotencyKeyHeader    = "Idempotency-Key"
+	// #nosec G101 - Not an actual secret, just the name of a header: https://docs.gitlab.com/user/project/integrations/webhooks/#create-a-webhook
+	defaultGitlabSecretTokenHeader = "X-Gitlab-Token"
```

Comment:
```suggestion
	defaultGitLabSecretTokenHeader = "X-Gitlab-Token"
```

Let's case `GitLab` appropriately. What usually happens to me is I start mixing them up and it gets confusing. I suggested a case change above too, but if we want to role that with another update, that'd be fine. It's a nit.